core/server/middleware/error-handler.js

```diff
@@ -0,0 +1,109 @@
+var _ = require('lodash'),
```

Comment:
yes totally agree 👍 
as you said, this is just the first refactoring of having cleaner code and especially cleaner handling!
later it's easy to just split both error handlers.